app/models/event_log.rb

```diff
@@ -11,7 +11,7 @@ class EventLog < ActiveRecord::Base
   UNSUCCESSFUL_LOGIN = "Unsuccessful login"
   SUSPENDED_ACCOUNT_AUTHENTICATED_LOGIN = "Unsuccessful login attempt to a suspended account, with the correct username and password"
   UNSUCCESSFUL_PASSPHRASE_CHANGE = "Unsuccessful passphrase change"
-  EMAIL_CHANGE_INITIATIED = "Email change initiated"
+  EMAIL_CHANGE_INITIATIED = "Email changed"
```

Comment:
Yes, it is tricky. You're right and I've added another event for the user flow for sake of clarity:
```
Email change confirmed
Email change initiated by Vinay Patel from abc@example.com to xyz@example.com
```